src/main/java/gov/cms/madie/models/measure/Group.java

```diff
@@ -55,4 +55,6 @@ public class Group implements GroupScoringPopulation {
 
   @Min(1)
   private Integer scoringPrecision;
+
+  private String displayId;
```

Comment:
we might need to add validator as well or is it going to be new ticket?

Reply:
@adongare The validator is in measure-service, when group is added or updated.